src/ui/components/NetworkMonitor/RequestDetails.tsx

```diff
@@ -63,6 +64,15 @@ const Cookies = ({ request }: { request: RequestSummary }) => {
   );
 };
 
+const StackTrace = () => {
+  return (
+    <div>
+      <h1 className="p-3 font-bold">Stack Trace</h1>
+      <Frames panel="debugger" />
```

Comment:
I also tried `webconsole` instead of `debugger` here because we check for that in a file, but changing it doesn't seem to do anything, so I'm just leaving as is for now.

Reply:
that's weird

Reply:
Lets wrap this in a div so we can add some padding lol. Amazing that this just works though. Also, where do you pass the frames in? If you select different requests you should have different frames?

In hindsight - i think that's the interesting thing. At this point we probably have the network requests in redux and we probably should have a dispatch that fetches the frames and saves them for the request in redux as well so we do not re-fetch them.

Reply:
Well, when we jump to a network request we are jumping to a pause with frames, so we are already fetching the frames (because that's what is displayed in the Pause panel). If the fetch is expensive, I agree that we should be caching it, but this change shouldn't be adding any additional load to what was already there, it's just displaying the information in another place. You can see this by opening the pause panel right now and jumping forward and backward to requests in the network monitor.